src/client/AutherClient.js

```diff
@@ -0,0 +1,52 @@
+export class AutherClient {
+  #location = window.location
+
+  constructor ({ redirectUri, autherUrl, http }) {
+    this.redirectUri = redirectUri
```

Comment:
why `uri` and not `url`?

Reply:
When an Auther returns with an authorization key, It need to know the full path, including anchors, in order to redirect correctly. Usually in such cases all call `URI`

Reply:
URL includes anchors 🤔 
https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#basics_anatomy_of_a_url